packages/web/src/views/launchpad/launchpad-detail/components/launchpad-pool-list/launchpad-pool-list-card/LaunchpadPoolListCard.tsx

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable @next/next/no-img-element */
+import React from "react";
+import { cx } from "@emotion/css";
+import { useAtomValue } from "jotai";
+
+import { LaunchpadState } from "@states/index";
+import { getTierDuration } from "@utils/launchpad-get-tier-number";
+import { LaunchpadPoolModel } from "@models/launchpad";
+import { ProjectRewardInfoModel } from "@views/launchpad/launchpad-detail/LaunchpadDetail";
+import { toNumberFormat } from "@utils/number-utils";
+
+import { Divider } from "@components/common/divider/divider";
+import { CardWrapper } from "./LaunchpadPoolListCard.styles";
+import LaunchpadPoolTierChip from "@views/launchpad/components/launchpad-pool-tier-chip/LaunchpadPoolTierChip";
+import DepositConditionsTooltip from "@components/common/launchpad-tooltip/deposit-conditions-tooltip/DepositConditionsTooltip";
+import MissingLogo from "@components/common/missing-logo/MissingLogo";
+import { LAUNCHPAD_DEFAULT_DEPOSIT_TOKEN } from "@common/values/token-constant";
+import { formatRate } from "@utils/new-number-utils";
+
+interface LaunchpadPoolListCardProps {
+  data: LaunchpadPoolModel;
+  idx: number;
+  rewardInfo: ProjectRewardInfoModel;
+
+  selectProjectPool: (poolId: number) => void;
+}
+
+const LaunchpadPoolListCard: React.FC<LaunchpadPoolListCardProps> = ({
+  data,
+  idx,
+  rewardInfo,
+  selectProjectPool,
+}) => {
+  const isShowConditionTooltip = useAtomValue(
+    LaunchpadState.isShowConditionTooltip,
+  );
+  const currentPoolId = useAtomValue(LaunchpadState.selectLaunchpadPool);
+
+  const isActiveCard = React.useMemo(() => {
+    return currentPoolId === data.id;
+  }, [currentPoolId, data.id]);
+
+  const getClaimableDays = (poolTier: string) => {
+    switch (poolTier) {
+      case "TIER180":
+        return 14;
+      case "TIER90":
+        return 7;
+      case "TIER30":
+        return 3;
+      default:
+        return 0;
+    }
```

Comment:
If this data is to be managed, it seems like a good idea to manage it in a constant file.